docs/user/device-groups.md

```diff
@@ -0,0 +1,47 @@
+---
+navTitle: Device Groups
+---
+
+# Device Groups
+
+
+**Navigation**: `Team > Application > Device Groups`
+
+<img src="./images/ui-device-groups.png" width="100%" />
+
+When managing many devices that are intended to run the same [snapshot](./snapshots.md), Device Groups allow you
+to organise your devices into logical groups.
+These groups can then be set as the target of a [DevOps Pipelines](./devops-pipelines.md).
+This greatly simplifying deployments of the same configuration to one or even hundreds of devices with a single click.
+
+The following requirements apply:
+
+- FlowFuse 1.15+ running with an EE license
```

Comment:
@Steve-Mcl There's no EE license, it doesn't exist. It's the Enterprise tier.

Reply:
@ZJvandeWeg this is coflicting with @knolleary suggestion: https://github.com/FlowFuse/flowfuse/pull/3318#discussion_r1449053434

Please advise.

Reply:
@Steve-Mcl it isn't conflicting - its an improved iteration. 

```suggestion
- FlowFuse 1.15+ Enterprise Tier
```